tests/main/cloud-init/task.yaml

```diff
@@ -17,9 +17,7 @@ execute: |
     if [[ ! -e /run/cloud-init/instance-data.json ]]; then
         echo "cloud-init instance data is required to execute the test"
 
-        if [[ "$SPREAD_SYSTEM" == ubuntu-* &&
-              "$SPREAD_SYSTEM" != ubuntu-14.04-* &&
-              "$SPREAD_SYSTEM" != ubuntu-core-20-* ]]; then
+        if os.query is-ubuntu && not os.query is-trusty && not os.query is-core20; then
```

Comment:
hmm but this test should be run on uc20 in GCE?

Reply:
Well, reading the code there is a comment after that line which says 
# we expect the test to run on all Ubuntu images, excluding 14.04/core20
This is in case: [[ ! -e /run/cloud-init/instance-data.json ]]

Reply:
I guess my point is that I think we _should_ now have `/run/cloud-init/instance-data.json` on uc20, so if that file doesn't exist and we are on uc20 then the test should fail.

Reply:
got it, thanks for clarifying this